presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1180,6 +1181,11 @@ public SystemSessionProperties(
                                 "To make it work, the connector needs to guarantee and expose the data properties of the underlying table.",
                         featuresConfig.isPreferMergeJoin(),
                         true),
+                booleanProperty(
+                        SEGMENTED_AGGREGATION_ENABLED,
+                        "Enable segmented aggregation.",
+                        false,
```

Comment:
Since after the worker side implementation is complete, we can remove this session property, that's why I didn't create the config accordingly

Reply:
I think it's probably good to have a config because we might continue to use it even after the worker side implementation is complete e.g. if there's a bug somewhere, it would be good to have a shutoff switch.